### DISCUSSION TITLE
[Feature request] popup window for button options on android

### DISCUSSION THREAD

**Discussion by kulizivo** [chrisgurney/obsidian-note-toolbar#109] [Ideas]
Add popup window on android floating button as-well instead of the system options panel

**Comment by chrisgurney** [comment]
@kulizivo thanks for your idea!

I'm not sure I understand though. Can you provide more details? If you think a video of another app would be helpful, I would appreciate that.

**Comment by chrisgurney** [comment]
With 1.9.6, at least on iOS, you can now _long press_ to get the options menu, and _tap_ to get the toolbar items. 

Not sure if that's what you meant? If so can you confirm it works on Android? Thanks!

**Comment by kulizivo** [comment]
![image](https://github.com/user-attachments/assets/a07635bc-d067-4b63-a139-9985c532685b)
![image](https://github.com/user-attachments/assets/b7be8c95-63e7-4755-9135-f2854e8711eb)

@chrisgurney sorry for the late response school and all.
So as you can see on android the menu covers the bottom and increases in height with options.
now what i wanted was the behavior you set for PC as shown in the smaller screenshot that i took from your recent release 1.9.6

so in case its still confusing i want the floating button options menu on android to behave like the one on PC

**Comment by kulizivo** [comment]
or a nicer way to put it is i also want a floating menu to go with the floating button

**Comment by chrisgurney** [comment]
@kulizivo OK, I understand now, thanks for elaborating.

As I inspect what's going on, Obsidian switches to the other style of menu when it detects that it's running on a phone (i.e., the body has the `.is-phone` class). There's style changes, and I _believe_ JavaScript changes at play here. That said, I don't think this is a trivial change, and it could very well break in the future. 

The API for menus is very limited as well, requiring a hack just to get some of the changes I previously implemented working well.

I'd be curious if you were to take a look for plugins, themes, or CSS snippets that might achieve what you're after here.

**Comment by kulizivo** [comment]
@chrisgurney Okay ill try... though I'm not that good at coding so it will definitely be trial and error.
Either way thanks for the feed back
